### ISSUE TITLE
LinkBubbleMenu not displayed if your Editor is inside a MUI Dialog modal

### ISSUE THREAD

**Issue by alaa-paramount** [sjdemartini/mui-tiptap#265]
### Is your feature request related to a problem? Please describe.

LinkBubbleMenu is using MuiPopper wrapper, the problem is with the zIndex value between 3-4 by default (bug?)

### Describe the solution you'd like

My expectation is to see the Link bubble menu from anyway, even when executing it inside a Dialog modal (but we see nothing on click)

### Describe alternatives you've considered

My temp solution is to add this code to my createTheme to make it work:
createTheme({
 MuiPopper: {
      styleOverrides: {
        root: {
          // a fix for the tiptap links inside a modal
          zIndex: "1300 !important"
        }
      }
    }
})

**Comment by mohanlokesh** [comment]
I have used like this in createTheme as alternative. 

```
MuiPopper: {
        styleOverrides: {
          root: {
            '&.MuiTiptap-ControlledBubbleMenu-root': {
              zIndex: 'var(--mui-zIndex-drawer)'
            },
            '&.MuiPopper-root[class*="ColorPickerPopper-root"]': {
              zIndex: 'var(--mui-zIndex-drawer)'
            }
          }
        }
```

**Comment by sjdemartini** [comment]
There is another approach that's been discussed before here to avoid this problem https://github.com/sjdemartini/mui-tiptap/issues/206#issuecomment-2133851815

But I'll look into changing the default zIndex style too.

**Comment by sjdemartini** [comment]
I've updated the default z-index of the bubble menus and poppers to use `theme.zIndex.tooltip` to fix this problem, released in [v1.14.0](https://github.com/sjdemartini/mui-tiptap/releases/tag/v1.14.0).

**Comment by alaa-paramount** [comment]
@sjdemartini Great, thanks!
